### PR TITLE
NEW Add Swift Mailer to send mails

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

**Comment by frederic34** [comment]
http://www.dolibarr.fr/forum/3-installation/54819-mail-pieces-jointes-n-arrivents-pas-sur-outlook?start=30#70676

**Comment by marcosgdf** [comment]
Related #4760 

I think it should replace the actual class or we are going to have duplicated code

File: htdocs/admin/mails.php [hunk 1]
```diff
@@ -705,9 +720,6 @@ function initfields()
 	{
 		print load_fiche_titre($langs->trans("DoTestServerAvailability"));
 
-		// If we use SSL/TLS
```
Comment (eldy) [inline htdocs/admin/mails.php:708]: Why removing this ?

Comment (frederic34) [inline htdocs/admin/mails.php:708]: Moved in Cmailfile.class.php in function check_server_port

File: COPYRIGHT [hunk 2]
```diff
@@ -30,6 +30,7 @@ PHPPrintIPP            1.3           GPL-2+                      Yes
 Restler                3.0           LGPL-3+                     Yes             Library to develop REST Web services
 TCPDF                  6.2.12        LGPL-3+                     Yes             PDF generation
 TCPDI                  1.0.0         LGPL-3+ / Apache 2.0        Yes             FPDI replacement
+Swift Mailer           5.4.2-DEV     MIT license                 Yes             Comprehensive mailing tools for PHP
```
Comment (marcosgdf) [inline COPYRIGHT:33]: I really hope this is a mistake and you've not added a development version…

Comment (rdoursenaud) [reply COPYRIGHT:33]: The file `htdocs/includes/swiftmaile/lib/classes/Swift.php` indicates version `6.0.2`.
@frederic34 :
If this is the correct version, the `COPYRIGHT` file should be updated.
Please also consider adding swiftmailer via composer #3351 to simplify dependencies maintenance : https://packagist.org/packages/swiftmailer/swiftmailer
Please also see #10203
Thanks !

**Pull request merged**